plugins/jobs/src/js/components/JobsBreadcrumbs.d.ts

```diff
@@ -1,10 +1,17 @@
 import { Component } from "react";
 import JobTree from "#SRC/js/structs/JobTree";
 import Job from "#SRC/js/structs/Job";
+import { JobTaskStatus } from "#PLUGINS/jobs/src/js/types/JobTaskStatus";
+import { Job as JobType } from "#PLUGINS/jobs/src/js/types/Job";
+
+interface JobSchedule {
+  enabled: boolean;
+  cron: string;
+}
 
 export default class JobsBreadcrumbs extends Component {
   props: {
-    tree: JobTree;
-    item: JobTree | Job;
+    item: Partial<JobType>;
```

Comment:
I don' understand why we need partial here. If you say it is a Job, will the compiler shout at you because you are not properly passing the full Job?